setup.py

```diff
@@ -15,7 +15,7 @@
     "py-eth-sig-utils>=0.3.0",
     "typing-extensions==3.10.0.0; python_version < '3.8'",
     "requests>=2",
-    "web3>=5.18,<=5.19",
+    "web3==5.23.0",
```

Comment:
The code previously was specifying that `web3` should be either `5.18.x` or `5.19.x`. Is it totally safe to update this?

Reply:
Yes this update was part of this change https://github.com/gnosis/gnosis-py/pull/43 which was released here https://github.com/gnosis/gnosis-py/releases/tag/v3.2.0

So the project itself requires `5.23.0` as a dependency but this dependency has an incompatible version being exposed to projects who use `gnosis-py`. So:
- For gnosis-py `web3 5.23.0` is already set and being used
- For projects that depend on the gnosis-py they fetch `web>=5.18,<=5.19` which is incompatible with the latest release (example: https://github.com/gnosis/safe-config-service/pull/198 – check actions tab for this change)